app/controllers/risks_controller.rb

```diff
@@ -27,11 +27,17 @@ def summary
     render 'summary/risk'
   end
 
+  def confirm
+    workflow = DocumentWorkflow.new(risk)
```

Comment:
if we're not checking for the outcome of this, it could be a one-liner:

`DocumentWorkflow.new(risk).update_status(:complete)`